core/src/main/java/org/infinispan/interceptors/impl/EntryWrappingInterceptor.java

```diff
@@ -254,21 +263,43 @@ public Object visitGetAllCommand(InvocationContext ctx, GetAllCommand command) t
             }
          }
          return delayedValue(stage.freeze(), rv, t);
-      });
+      };
+
+      if (aggregateCompletionStage != null) {
+         return asyncValue(aggregateCompletionStage.freeze()).andHandle(ctx, command, handleFunction);
```

Comment:
isn't the `invokeNext` missing?

Reply:
Yes, oddly I actually fixed this last night. Thought I pushed the update, but I guess not.

Reply:
fixed in the wrong PR https://github.com/infinispan/infinispan/pull/7722/commits/e40b3f37121be2447c82453c6c8f7df7e786f7ce#diff-d7e74cc6236d782d324013d36fc08314R269 :)